app/actors/sufia/create_with_files_actor.rb

```diff
@@ -42,5 +43,13 @@ def uploaded_files
         return [] if uploaded_file_ids.empty?
         @uploaded_files ||= UploadedFile.find(uploaded_file_ids)
       end
+
+      def set_publication_state
+        self.curation_concern.state = unpublished if Flipflop.enable_mediated_deposit?
```

Comment:
This is very minor, but would you consider:

1. renaming the new methods;
1. removing the explicit `self`; and 
1. reworking this method with a guard clause?

```ruby
def mark_inactive
  return unless Flipflop.enable_mediated_deposit?
  curation_concern.state = inactive_uri
end

def inactive_uri
  ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive')
end
```